common/middleware/auth.js

```diff
@@ -2,7 +2,9 @@ const passport = require('passport')
 const { Strategy } = require('passport-oauth2')
 const refresh = require('passport-oauth2-refresh')
 const jwtDecode = require('jwt-decode')
+// eslint-disable-next-line import/no-unresolved
```

Comment:
I would remove these `eslint-disable`s as we should be able to resolve the dependencies